src/blocks/kdeconnect.rs

```diff
@@ -129,6 +137,22 @@ pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
                     }
                 }
 
+                let network = device.network().await;
```

Comment:
You can do something like this so you don't have to recall what .0 and .1 are in the tuple:

```rust
let (cellular_network_type, cellular_network_strength) = device.network().await;
```

Reply:
I show how you can do this in the diff above.